apps/web/.env.example

```
@@ -4,5 +4,5 @@ VITE_FIREBASE_PROJECT_ID=
 VITE_FIREBASE_APP_ID=
 VITE_API_URL=http://localhost:8080
 VITE_ALLOWED_HOSTS=localhost
-VITE_AVATAR_BASE_URL=https://lumo-file-content.s3.eu-north-1.amazonaws.com
+VITE_AVATAR_BASE_PATH=/avatars
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_AVATAR_BASE_PATH key should go before the VITE_FIREBASE_API_KEY key

(UnorderedKey)